### PR TITLE
Add Old Buddy color scheme optimized for PowerShell code

### PR DESCRIPTION
**Pull request by guillermooo**
This PR also sets the Old Buddy color scheme as the default color scheme for PowerShell files. Users can still override this. Not sure whether it's a good idea, though it'd sure raise the color scheme's profile :)

### CODE REVIEW

**Comment by vors** [comment]
I would not change default color schema. People are sensitive about such things.

**Comment by guillermooo** [comment]
Agreed. I will remove the setting.

File: Color Scheme - Old Buddy/Old Buddy.tmTheme [hunk 1]
```diff
@@ -0,0 +1,286 @@
...
+			<key>name</key>
+			<string>Function name</string>
+			<key>scope</key>
+			<string>entity.name.function, support.function.any-method</string>
+			<key>settings</key>
+			<dict>
+				<key>fontStyle</key>
+				<string>bold</string>
+				<key>foreground</key>
+				<string>#0000A2</string>
```
Comment (vors) [inline Color Scheme - Old Buddy/Old Buddy.tmTheme]: ISE uses purple

**Comment by vors** [comment]
In general, feels like it uses too much bold. Text looks heavier, then it should, imo. But I'm a noobe in color schema. @bcrotty ?

**Comment by vors** [comment]
BTW, I think it's awesome anyway.
Feel free to merge.

**Comment by bcrotty** [comment]
@vors I think it would be helpful to include this, but it might be easier to find/understand if we name it something like "PowerShell ISE". There are still quite a few differences from ISE though. I'd want to keep tweaking these, but some might be syntax differences as well.

**Comment by guillermooo** [comment]
I've fixed a few things. I think this ready for a first version.

Re. name. I don't know. Maybe using 'PowerShell ISE' is a problem. Also, as you say, it isn't exactly the same scheme. 'Old Buddy' alludes to PSISE, but it's certainly obscure. I'm not sure we should worry too much about the name, though. Maybe if we advertise as it as 'based on PowerShell ISE colors' in the readme, changelog, etc. and include a screenshot it'll be enough.

I'm open to changing the name; I just don't know if 'PowerShell ISE' is the right one due to potential legal concerns or misinformation.

**Comment by guillermooo** [comment]
ps-console

**Comment by guillermooo** [comment]
Ooops. That screenshot is about funspace. #92

**Comment by vors** [comment]
I asked our PM for powershell_ise and he recommend use something like "PowerShell ISE-like", but not "PowerShell ISE" to avoid confusing and questions from LCA.

**Comment by guillermooo** [comment]
Thanks for inquiring, @vors.

"PowerShell ISE-like" won't be too catchy... How about "PS Standard"?

PS Basic
PS Original
PS Light
...

**Comment by vors** [comment]
Currently in preferences schema is under
Color Schemas -> PowerShell -> Old Buddy -> Old Buddy light

Subfolder Old Buddy is unnecessary.
Also, because PowerShell already showed up in this path, we can use "ISE inspired" as a name (dropping PowerShell word).

**Comment by vors** [comment]
Wait, it's actually even longer:
image

**Comment by guillermooo** [comment]
Hm. I think you have a level too many. The package's top-level dir should be named *PowerShell*, BTW. Otherwise I don't think the Python code will work.

I'll move the theme out of the Support folder.

**Comment by vors** [comment]
Yeah, it's my dev installation and name is different historically.

**Comment by guillermooo** [comment]
Still not happy with the name, though. If you look at the default names for color schemes, they are all pretty fancy. 'ISE Inspired' looks too dry by comparison.

**Comment by guillermooo** [comment]
ISE New or ISE Modern could work for me.

Also ISE Nostalgia, but that's perhaps too fancy?

**Comment by vors** [comment]
I like 'ISE Nostalgia'

**Comment by vors** [comment]
Preview for references

image

**Comment by guillermooo** [comment]
@vors I'll try to change the color scheme's name as discussed and then merge this.

**Comment by guillermooo** [comment]
Closing this one and reopening new PR.

**Pull request closed without merging**